src/pytypes/onefuzztypes/models.py

```diff
@@ -273,6 +273,7 @@ class ADOTemplate(BaseModel):
     unique_fields: List[str]
     comment: Optional[str]
     ado_fields: Dict[str, str]
+    ado_duplicate_fields: Optional[Dict[str, str]]
```

Comment:
can you updates the examples to includes the new field
docs\notifications\ado.md
contrib\onefuzz-job-azure-devops-pipeline\ado-work-items.json